subsys/usb/usb_device.c

```diff
@@ -572,6 +615,11 @@ static bool usb_set_interface(u8_t iface, u8_t alt_setting)
 	u8_t cur_iface = 0xFF;
 	bool found = false;
 
+	if ((oldIface != 0xFF) && (oldAlt_setting != 0xFF))
```

Comment:
Can we have set_interface called twice with the same parameters?

Reply:
@finikorg I have not seen in the standard any restriction to do so.